core/pom.xml

```diff
@@ -50,7 +50,7 @@ Bundle-SymbolicName: com.adobe.aem.aem-modernize-tools.core
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
+            <!-- <plugin>
```

Comment:
This plugin is required to be used in order to validate that the versions are being set correctly based on changes.

Reply:
Uncommented this code and bumped version for all packages.